cli/src/pcluster/validators/slurm_settings_validator.py

```diff
@@ -197,3 +197,14 @@ def _validate(self, database, external_slurmdbd):
                 "Database and ExternalSlurmdbd cannot be defined at the same time within SlurmSettings.",
                 FailureLevel.ERROR,
             )
+
+
+class ExternalSlurmdbdTrafficNotEncrypted(Validator):
+    """Inform users about unencrypted connections."""
+
+    def _validate(self, external_slurmdbd):
+        if external_slurmdbd is not None:
+            self._add_failure(
+                "Traffic between ParallelCluster and the external Slurmdbd is not encrypted.",
```

Comment:
Do we need to get this message approved?

Reply:
Done